src/App.tsx

```diff
@@ -1,57 +1,93 @@
-import "./App.css";
+import { useState } from "react";
+import Banklink from "./components/BankLink";
+import SideNav from "./components/SideNav";
+import { BankData } from "../types";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+// const data: BankData[] = new Array(142).fill({ name: "test", link: "test" });
+const data: BankData[] = [
+  {
+    name: "Central Bank of Kenya",
+    link: "https://www.centralbank.go.ke/policy-procedures/legislation-and-guidelines/circulars/",
+  },
+  {
+    name: "Central Bank of Nigeria",
+    link: "https://www.cbn.gov.ng/documents/circulars.asp?beginrec=1&endrec=20&keyword=&from=&tod=",
+  },
+  {
+    name: "Bank of Zambia",
+    link: "https://www.boz.zm/circulars.htm",
+  },
+  {
+    name: "Banco de Mozambique",
+    link: "https://www.bancomoc.mz/en/media/communications/?query=&page=1",
+  },
+  {
+    name: "Reserve Bank of Malawi",
+    link: "https://www.rbm.mw/MediaCenter/PressReleases/",
+  },
+  {
+    name: "National Bank of Rwanda",
+    link: "https://www.bnr.rw/news-publications/news/news-press-release/",
+  },
+  {
+    name: "Bank of Tanzania",
+    link: "https://www.bot.go.tz/Publications/Filter/39",
+  },
+];
+
+const itemsPerPage = 7;
 
 function App() {
```

Comment:
Slight nit pick. Since the other components follow the `export default function Component` can we do the same here